packaging/debian/control

```diff
@@ -1,13 +1,17 @@
 Source: cvmfs-config-osg
 Maintainer: Dave Dykstra <dwd@fnal.gov>
-Section: config
+Section: utils
 Priority: extra
-Standards-Version: 3.9.3.1
-Build-Depends: debhelper (>= 9) 
-Homepage: http://github.conf/opensciencegrid/cvmfs-config-osg
+Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9), dh-systemd
```

Comment:
This dh-systemd dependency is no longer necessary, correct?

Reply:
I think not, but I'm not 100% certain. The `rules` file contains
```
#!/usr/bin/make -f
%:
	dh $@
```
I think that, on systemd platforms, this expands to automatically performing substitutions in the `postinst/postrm` scripts. I think these are no-ops for this package.

So, it can probably be removed. But, so could `postinst` and `postrm` and maybe some part of the `rules`? I kinda like leaving the infrastructure in place, even if it's not presently doing much.

Reply:
I was almost persuaded by your response, but for some reason it is causing all the [OBS builds](https://build.opensuse.org/package/show/home:drdaved:cvmfs-contrib/cvmfs-config-osg) to fail.  It would make sense on the pre-systemd versions, and in any case we should remove it for that I think.  So please go ahead and take that out.

Reply:
OK. I eliminated the `dh-systemd` dependence and decided to also eliminate `postinst` and `postrm` since they are now unnecessary.